src/pages/Profile/Profile.page.tsx

```
@@ -1,8 +1,18 @@
-        <Typography>Profile</Typography>
-    </BaseLayout>
-);
+import { StyledBox } from './Profile.styles';
+
+export const Profile = () => {
+    const { userName } = useParams();
+
+    return (
+        <BaseLayout>
+            <StyledBox>
+                <ProfileInfo userName={userName!} />
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)